code/src/main/java/com/onlinecontacttracing/servlets/get-data/CalendarDataForPositiveUser.java

```diff
@@ -1,16 +1,74 @@
 package com.onlinecontacttracing.authentication;
 
 import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import com.onlinecontacttracing.storage.Constants;
+import com.onlinecontacttracing.storage.PositiveUserContacts;
+import com.google.api.services.calendar.model.EventAttendee;
+import java.util.List;
+import java.util.Collections;
+import java.util.Optional;
+import com.googlecode.objectify.Objectify;
 
 class CalendarDataForPositiveUser implements Runnable {
+
+  private static final String APPLICATION_NAME = "Google Calendar API Java Quickstart";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+  private final String userId;
```

Comment:
nit: please always order arguments, fields and assignments the same way - a constructor should have all of these in the same order.

Reply:
my b